configure.ac

```diff
@@ -2,8 +2,11 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.69])
-AC_INIT([abcip], [2.2.1], [crc181@gmail.com])
+AC_INIT([abcip],
```

Comment:
So how do I assign a version with this change?  I want to set it to 2.3.0 after merge.